### DISCUSSION TITLE
GitHub shows false warnings

### DISCUSSION THREAD

**Discussion by richelbilderbeek** [community/community#18643] [Other Feature Feedback, Questions, & Ideas]
Dear GitHub,

First, I am a big fan of GitHub! 

I do notice something since june 13th, which are warnings such as these displayed in the top:

![Screenshot from 2022-06-15 09-12-11](https://user-images.githubusercontent.com/2098230/173765388-9277daa9-fc23-4e85-b496-b93d419a7251.png)

Indeed, the file `gepi22326-fig-0002-m.jpg` is not there, as shown from the locally cloned repo:

```
richel@N141CU:~/GitHubs/richel$ git show -- gepi22326-fig-0002-m.jpg
```

The file `gepi22326-fig-0002-m.jpg` *was* there once, as shown from the locally cloned repo:

```
richel@N141CU:~/GitHubs/richel$ git log -- gepi22326-fig-0002-m.jpg
commit 90824f7e9d55cfe790932687c4e077f58cc0c53d
Author: richelbilderbeek <richel@richelbilderbeek.nl>
Date:   Wed Feb 2 08:48:48 2022 +0100

    Cleaned up

commit 6c89b8d4d2fe0b6c0021611b0325cb1a14552089
Author: richelbilderbeek <richel@richelbilderbeek.nl>
Date:   Wed Apr 28 12:40:43 2021 +0200

    Add Quick et al., 2017 picture
```

Removing that warning would make GitHub as cool as can be again :-)

Note that, when clicking 'I have done a search for similar discussions*', the warning is shown there as well:

![Screenshot from 2022-06-15 09-16-45](https://user-images.githubusercontent.com/2098230/173766444-6f6717b9-158c-461e-8708-5e449ead6548.png)

Good luck fixing it, I'd happily help! Cheers, Richel Bilderbeek

**Comment by MathiasReker** [comment]
Hello. I was reading about this a few days ago. This is already in the pipeline and should be fixed very soon.

**Reply by richelbilderbeek** [reply]
Awesome, thanks!
